frontend/src/metabase/query_builder/components/filters/modals/BulkFilterItem/BulkFilterItem.tsx

```diff
@@ -174,13 +174,17 @@ const getNewFilter = (query: StructuredQuery, dimension: Dimension): Filter => {
     useDefaultOperator: !isBooleanField,
   });
 
-  const isNumericField = isNumber(field);
+  const isNumericField = isNumber(field) && field.has_field_values !== "list";
```

Comment:
unrelated: we also don't want to default to `between` if has_field_values is set to `list`, because we assume the user wants to see a list of options